infrastructure/main.tf

```diff
@@ -18,6 +18,10 @@ provider "aws" {
   region = "eu-west-2"
 }
 
+provider "aws" {
```

Comment:
The `aws` provider version would be better locked to `5` e.g. `version ="~> 5.0"`

Reply:
depreciated in place of 'required_providers' in the global terraform block, which makes sure all imported aws providers adhere to one version